.github/workflows/test.yml

```diff
@@ -55,6 +55,27 @@ jobs:
         if: runner.os == 'Linux' && matrix.app_version == 'stable'
         env:
           NEOVIM_PATH: ${{ steps.vim.outputs.executable }}
+      - name: Set up Lua
+        uses: leafo/gh-actions-lua@v9
+        with:
+          luaVersion: "luajit-2.1.0-beta3"
+        if: runner.os == 'Linux' && matrix.app_version == 'stable'
+      - name: Set up Luarocks
+        uses: leafo/gh-actions-luarocks@v4
+        with:
+          luaVersion: "5.1"
+        if: runner.os == 'Linux' && matrix.app_version == 'stable'
```

Comment:
@pokey are you aware of a way to avoid having to specify `if: runner.os == 'Linux' && matrix.app_version == 'stable'` for multiple steps as seen here and below?

Reply:
Seems possible by using a `uses: composite` entry and then have a set of additional `steps:` entries, so will give it a try.

Reply:
Turns out you can only use composite if you break it out into a separate file, so open to other suggestions.

Reply:
`uses: composite` with a separate `test_neovim_lua.yml` file might actually be nice?

Reply:
Ya I think it would be nicer/cleaner in a separate file personally. Can do it if you want

Reply:
Also is `using: composite` my bad

Reply:
This is done. I think the main downside is that it seems it has to go inside .github/actions/<something>/action.yml and you can't just name it whatever.yml. But otherwise is cleaner imo.